mkdocs/docs/experiments/categories/azure/azure-instance-stop/chaos-interval.yaml

```diff
@@ -22,3 +22,6 @@ spec:
           value: 'instance-01,instance-02'
         - name: RESOURCE_GROUP
           value: '<resource group of AZURE_INSTANCE_NAME>'
+        # accepts enable/disable value. default is disable
```

Comment:
It is not a mandatory field so we will provide it in https://github.com/litmuschaos/litmus/blob/a0ac61ed1507fea458e8d73e9bee04578a86f06e/mkdocs/docs/experiments/categories/azure/azure-instance-stop/azure-scale-set-instance.yaml file only

Reply:
Should I mention it in the Optional Fields section?

Reply:
I mean we can remove SCALE_SET ENV from here and from other files except the `azure-scale-set-instance-yaml` as it is an optional field